ersilia/utils/docker.py

```diff
@@ -320,7 +320,26 @@ def container_peak(self, model_id):
         except Exception as e:
             print(f"An error occurred: {e}")
```

Comment:
Can we also remove this print statement, please? Let's replace it with a debug log.

Reply:
Definitely!
I'll get back on changes in a couple of hours

Reply:
@DhanshreeA I'm done implementing changes @DhanshreeA